target/linux/mediatek/dts/mt7981b-nradio-wt9109-second-system.dts

```diff
@@ -0,0 +1,270 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+/dts-v1/;
+
+#include "mt7981.dtsi"
+
+/ {
+	model = "NRadio C8-668GL (Second System)";
+	compatible = "nradio,hcmt7981-emmc", "mediatek,mt7981-emmc-rfb";

```

Comment:
This compatible does not match the model name.

Reply:
dts filename: `nradio-wt9109-second-system`
dts model: `NRadio C8-668GL (Second System)`
dts compatible: `nradio,hcmt7981-emmc`
image definition: `nradio_wt9109_second_system`

Reply:
> This compatible does not match the model name.

Initially, the naming conventions were based on the official system, with some adjustments made for better readability. After careful consideration, I decided to adopt a naming style like `nradio_c8-668gl-second-system`.  
   - **Official naming example:**  
     ```
     compatible = "HCMT7981-EMMC", "mediatek,mt7981-emmc-rfb";
     model = "HC-WT9109";
     ```
   - **Revised naming scheme:**  
     ```
     DTS filename: nradio-c8-668gl-second-system  
     DTS model: NRadio C8-668GL (Second System)  
     DTS compatible: nradio,c8-668gl-second-system  
     Image definition: nradio_c8-668gl-second-system  
     ```

Reply:
Please use the same model name for all of them.

Reply:
"mediatek,mt7981-emmc-rfb"  don't need.